RemoteHealthcare/SimulatorBike.cs

```diff
@@ -125,15 +112,12 @@ public static byte[] generateAPage(byte pagenumber)
         }
     }
 
-    internal class FakeBikeData : BikeData
-    {
-        public byte[] Data { get; set; }
-        public string ServiceName { get; set; }
-
-        public FakeBikeData(byte[] data, string serviceName) : base(new avansBikeData())
+    internal class FakeBikeData : avansBikeData
+    { 
+        public FakeBikeData(byte[] data, string serviceName)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)